Vagrantfile

```diff
@@ -20,16 +20,22 @@ DISTRO=$(lsb_release -is | tr '[:upper:]' '[:lower:]')
 CODENAME=$(lsb_release -cs)
 echo "deb http://repos.mesosphere.io/${DISTRO} ${CODENAME} main" | sudo tee /etc/apt/sources.list.d/mesosphere.list
 
-apt-get -q -y update
+apt-get -o Acquire::http::Timeout=1 -o Acquire::ftp::Timeout=1 -y update
```

Comment:
@erikdw You are right. This change will help speed up `vagrant up` not building docker Image as I previously stated. In terms of whether or not its dangerous, we should specify the version explicitly if we think we need it for mesos-storm. If that version cannot be found, then ` vagrant up` should error out. As per your recommendation, I will specify the specific version of mesos for installation.